plugins/analytics-module-ga/README.md

```diff
@@ -0,0 +1,144 @@
+# Analytics Module: Google Analytics
+
+This plugin provides an opinionated implementation of the Backstage Analytics
+API for Google Analytics. Once installed and configured, analytics events will
+be sent to GA as your users navigate and use your Backstage instance.
+
+This plugin contains no other functionality.
+
+## Installation
+
+1. Install this plugin in your Backstage app:
+   `cd packages/app && yarn add @backstage/plugin-analytics-module-ga`
+2. Add the API implementation to your App and configure the plugin (see below).
+
+```tsx
+// packages/app/src/apis.ts
+import { analyticsApiRef, configApiRef } from '@backstage/core-plugin-api';
+import { GoogleAnalytics } from '@backstage/plugin-analytics-module-ga';
+
+export const apis: AnyApiFactory[] = [
+  // Instantiate and register the GA Analytics API Implementation.
+  createApiFactory({
+    api: analyticsApiRef,
+    deps: { configApi: configApiRef },
+    factory: ({ configApi }) => GoogleAnalytics.fromConfig(configApi),
+  }),
+];
+```
+
+## Configuration
+
+The following is the minimum configuration required to start sending analytics
+events to GA. All that's needed is your Universal Analytics tracking ID:
+
+```yaml
+# app-config.yaml
+app:
+  analytics:
+    provider: ga
+    ga:
+      trackingId: UA-0000000-0
+```
+
+### Plugin Analytics
+
+In order to be able to analyze usage of your Backstage instance _by plugin_, we
+strongly recommend configuring at least one custom dimension to capture Plugin
```

Comment:
It might be valuable to promote discoverability since we strongly recommend this usage by having a brief of a custom dimension or, at least, a link to the documentation for those unfamiliar with this concept.